build.ps1

```diff
@@ -198,17 +199,27 @@ function Build-CMakeProject
 
     $RuntimeBuildDir = Get-ProjectBuildDir $Arch 1
     $SwiftResourceDir = "${RuntimeBuildDir}\lib\swift"
-    $SwiftcFlags = @(
-      "-resource-dir $SwiftResourceDir",
-      "-L $SwiftResourceDir\windows",
-      "-vfsoverlay $RuntimeBuildDir\stdlib\windows-vfs-overlay.yaml",
-      "-g -debug-info-format=codeview",
-      "-Xlinker /INCREMENTAL:NO",
-      "-Xlinker /DEBUG",
-      "-Xlinker /OPT:REF",
-      "-Xlinker /OPT:ICF"
-    ) -Join " "
 
+    $SwiftArgs = [System.Collections.ArrayList]@()
+
+    if ($SwiftSDK -ne "") {
+      $SwiftArgs.Add("-sdk $SwiftSDK") | Out-Null
```

Comment:
I believe you can use a semicolon instead of ` | Out-Null` to suppress automatic expression output.

Reply:
If this were a regular array (not sure if ArrayList will behave the same), you could do:
`$SwftArgs += "-sdk $SwiftSDK"`
And it shouldn't print anything because it's more than a simple expression, there's an assignment

Reply:
🤯 Let me try that!

Reply:
No, that still prints the index :-(